common/src/main/java/org/apache/falcon/security/DefaultAuthorizationProvider.java

```diff
@@ -294,15 +296,16 @@ protected boolean isUserInAdminGroups(UserGroupInformation proxyUgi) {
 
     protected void authorizeEntityResource(UserGroupInformation authenticatedUGI,
                                            String entityName, String entityType,
-                                           String action)
+                                           String action, boolean entityTypeOptional)
         throws AuthorizationException, EntityNotRegisteredException {
-
-        Validate.notEmpty(entityType, "Entity type cannot be empty or null");
+        if (!entityTypeOptional) {
+            Validate.notEmpty(entityType, "Entity type cannot be empty or null");
+        }
         LOG.debug("Authorizing authenticatedUser={} against entity/instance action={}, "
                 + "entity name={}, entity type={}",
                 authenticatedUGI.getShortUserName(), action, entityName, entityType);
 
-        if (entityName != null) { // lifecycle actions
+        if (entityType != null && entityName != null) { // lifecycle actions
```

Comment:
This means that there wont be any authorization when entityType is null. Instead, you can change lines 301 to 303 as follows, 

if (entityTypeOptional && entityType == null) {
    Log(validation not required)
    return;
}

What do you think?